src/azure.rs

```diff
@@ -261,6 +261,16 @@ impl PartialEq for AzColor {
     }
 }
 
+impl AzColor {
+    pub fn new(r: AzFloat, g: AzFloat, b: AzFloat, a: AzFloat) -> AzColor {
+         AzColor { r: r, g: g, b: b, a: a }
+    }
+
+    pub fn hack(color: AzColor) -> AzColor {
```

Comment:
Legacy code calls converting the Color to AzColor and vice-versa. I will
try to remove them.

On Mon, Feb 9, 2015 at 12:38 PM, Martin Robinson <notifications@github.com>
wrote:

> In src/azure.rs
> <https://github.com/servo/rust-azure/pull/141#discussion_r24363008>:
>
> > @@ -261,6 +261,16 @@ impl PartialEq for AzColor {
> >      }
> >  }
> >
> > +impl AzColor {
> > +    pub fn new(r: AzFloat, g: AzFloat, b: AzFloat, a: AzFloat) -> AzColor {
> > +         AzColor { r: r, g: g, b: b, a: a }
> > +    }
> > +
> > +    pub fn hack(color: AzColor) -> AzColor {
>
> Why is it necessary? It's unclear why you need to create a new AzColor
> where it's used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/servo/rust-azure/pull/141/files#r24363008>.
>